src/States/babylon.depthCullingState.ts

```diff
@@ -38,19 +38,6 @@
             this._isZOffsetDirty = true;
         }
 
-        public get cullFace(): Nullable<number> {

```

Comment:
I'm fine with this PR. My only concern is this change in the public API
We need to keep cullFace property (and probably deduce it from frontFace)

Reply:
I can put this back. Is it necessary to set it at all? I can just leave it as default.

Reply:
it was as read/write property so wee need to keep it unaltered :D